core/Symbols.cc

```diff
@@ -2295,17 +2293,17 @@ uint32_t Method::methodShapeHash(const GlobalState &gs) const {
     return result;
 }
 
-vector<uint32_t> Method::methodArgumentHash(const GlobalState &gs) const {
-    vector<uint32_t> result;
-    result.reserve(arguments.size());
+// This has to match the implementation of ArgParsing::hashArgs
```

Comment:
I like this comment, but given that the current implementation doesn't actually match `ArgParsing::hashArgs`, it seems like we need to change one of the implementations or update the comment.

Reply:
I will take a look at this in a follow up (if it's broken because of this change, it was already broken on master)